src/wallet/generateFaucetWallet.ts

```diff
@@ -229,3 +286,9 @@ export function getFaucetUrl(client: Client) {
 }
 
 export default generateFaucetWallet
+
+const _private = {
+  FaucetNetwork,
+  getFaucetUrl,
+}
+export { _private }
```

Comment:
Why does `_private` have an underscore in it's name? (I'm not quite sure what that naming convention implies)

Reply:
It means it's a private variable. These methods are used for testing, so they need to be exported for that, but we don't want people to have access to it otherwise. Technically they still have access, but it's bad practice.